### ISSUE TITLE
Is there any way that it can be used to track objects detected by tensorflow object detection api?

### ISSUE THREAD

**Issue by gopalkalpande** [nwojke/deep_sort#168]

**Comment by anzy0621** [comment]
Yes, you have to modify the TensorFlow detection code in order to save the detection of bounding boxes in a textfile. Once you get the output "detections" textfile then you can just carry out this deep sort application as it is shown in the repo

**Comment by gopalkalpande** [comment]
Even I do create the text file it will serve the purpose of detecting the object in a video, but how can it be used to track the objects in a live video stream?

**Comment by rishabh700** [comment]
You can have a look at this repo https://github.com/Qidian213/deep_sort_yolov3
